src/utils/prim-lexing.fsi

```diff
@@ -65,6 +65,7 @@ type internal LexBuffer<'Char> =
     member IsPastEndOfStream: bool with get,set
 
```

Comment:
FromArryNoCopy should go here

Reply:
Notice that `FromArray` isn't exposed in this file, this is why I haven't added the extra method.

Reply:
ok thanks